src/System.Reflection.Emit.ILGeneration/tests/CustomAttributeBuilderTests.cs

```diff
@@ -876,17 +872,33 @@ public static void NullObjectInNamedProperties_ThrowsArgumentNullException()
             Assert.Throws<ArgumentNullException>("namedProperties[0]", () => new CustomAttributeBuilder(con, new object[0], namedProperties, new object[1], new FieldInfo[0], new object[0]));
         }
 
+        [Fact]
+        public static void IndexerInNamedProperties_ThrowsArgumentException()
```

Comment:
Test name should say "ThrowsCustomFormatException" rather than "ThrowsArgumentException"